x-pack/legacy/plugins/actions/server/actions_client.ts

```diff
@@ -92,6 +93,12 @@ export class ActionsClient {
     const validatedActionTypeConfig = validateConfig(actionType, config);
     const validatedActionTypeSecrets = validateSecrets(actionType, secrets);
 
+    try {
+      this.actionTypeRegistry.ensureActionTypeEnabled(actionTypeId);
```

Comment:
Sorry to repeat the point... but this too feels like it's missing tests

Reply:
ya, this needs a test, thx!

Reply:
test added